outfitter-mobile-app/App.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { StyleSheet, Text, View, TextInput, Button, ToastAndroid, TouchableHighlight } from 'react-native';
+import { Header } from 'react-native-elements';
+
+export default class App extends React.Component {
+  
+  constructor(props){
+    super(props);
+    this.login = this.login.bind(this);
+    this.state = {user: '', pass: ''};
+  }
+
+  login(){
+    ToastAndroid.show(`u: ${this.state.user} p: ${this.state.pass}`, ToastAndroid.SHORT);
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Header
+          placement="left"
+          leftComponent={{ icon: 'menu', color: '#fff' }}
+          centerComponent={{ text: 'Outfittr', style: { color: '#fff' } }}
+          />
+        <View style={styles.containerOther}>
+          <TextInput 
+            style={styles.loginBoxes} 
+            placeholder="username" 
+            placeholderTextColor="#90a4ae"
+            onChangeText={(user) => {this.setState({user})}}
+            />
+          <TextInput 
+            style={styles.loginBoxes} 
+            placeholder="password" 
+            textContentType="password"
+            secureTextEntry={true}
+            placeholderTextColor="#90a4ae"
+            onChangeText={(pass) => {this.setState({pass})}}
+            />
+          <TouchableHighlight
+            style={styles.loginButton}
+            onPress={this.login}
+            >
```

Comment:
Missing '/'

Reply:
that doesnt need one. there is a nested text element inside of it. ???

Reply:
Missed the closing tag when looking through. You're correct.